src/selectors/dataTypes/valueTypes/QName.js

```diff
@@ -1,23 +0,0 @@
-class QName {
```

Comment:
Leftover from implementing data types, until now not used anywhere, just some cleanup.